data/provisioners/bootstrap/aws/main.tf

```diff
@@ -9,26 +9,43 @@ terraform {
   required_providers {
     local    = "2.0.0"
     null     = "3.0.0"
-    aws      = "3.37.0"
+    aws      = "3.56.0"
     external = "2.0.0"
   }
 }
 
-module "vpc-and-vpn" {
-  source = "github.com/sighupio/fury-eks-installer//modules/vpc-and-vpn?ref=v1.10.0"
-
-  name                     = var.name
-  network_cidr             = var.network_cidr
-  public_subnetwork_cidrs  = var.public_subnetwork_cidrs
-  private_subnetwork_cidrs = var.private_subnetwork_cidrs
-  vpn_subnetwork_cidr      = var.vpn_subnetwork_cidr
-  vpn_port                 = var.vpn_port
-  vpn_instances            = var.vpn_instances
-  vpn_instance_type        = var.vpn_instance_type
-  vpn_instance_disk_size   = var.vpn_instance_disk_size
-  vpn_operator_name        = var.vpn_operator_name
-  vpn_dhparams_bits        = var.vpn_dhparams_bits
-  vpn_operator_cidrs       = var.vpn_operator_cidrs
-  vpn_ssh_users            = var.vpn_ssh_users
-  tags                     = var.tags
+module "vpc" {
+  source = "github.com/sighupio/fury-eks-installer//modules/vpc?ref=v2.0.0"
+
+  count = var.vpc_enabled ? 1 : 0
+
+  name         = var.name
+  network_cidr = var.network_cidr
+  tags         = var.tags
+
+  private_subnetwork_cidrs = var.vpc_private_subnetwork_cidrs
+  public_subnetwork_cidrs  = var.vpc_public_subnetwork_cidrs
+}
+
+module "vpn" {
+  source = "github.com/sighupio/fury-eks-installer//modules/vpn?ref=v2.0.0"
+
+  count = var.vpn_enabled ? 1 : 0
```

Comment:
with this approach, we are losing the capability of having more than one bastion. In production environments, we usually deploy 2 instances of the Bastion (VPN) to have HA.

Instead of hardcoding `1` we should use the value of `vpn_instances`

Reply:
hmm not really, the module hasn't fundamentally changed from the previous version, besides being split. we are still able to create multiple instances via the `vpn_instances` variable down below, pretty much as (I assume) we did in the past.
The semantic here is simply on/off. If it's on, then you can configure it exactly as we used to do in the past (that is, using multiple instances if deemed necessary).

Reply:
ohh I think I understand, the `count` affects only the count of `vpn` modules being created and not the number of instances, right?

Reply:
correct 👍